src/question.js

```diff
@@ -21,7 +21,7 @@ router.post('/check/:qno(\\d+)?', middleware.isAuthenticated, (req, res) => {
   var { qno } = req.params;
   const { answer } = req.body;
   const { lastQuestionAllowed } = req.user;
-  const {score} = req.user;
+  const { score } = req.user;
```

Comment:
I dont like unclean commits, but since anyway the commit history is not very clean, i dont mind merging this. But this is a bad practice. a commit must only do what it states